recipe/meta.yaml

```
@@ -1,17 +1,13 @@
 {% set name = "arcticdb" %}
-{% set version = "1.5.0" %}
+{% set version = "1.6.1-rc0" %}
 
 package:
   name: {{ name|lower }}
-  version: {{ version }}
+  version: 1.6.1rc0
```

This suggestion has to be committed with 2 other ones (1/3)

```suggestion
  # conda-forge does not support `-` as a separator, so we omit it.
  version: {{ version }}{{ version_suffix }}
```